app/src/components/UserForm.tsx

```diff
@@ -0,0 +1,182 @@
+"use client";
+
+interface UserFormProps {
+  formType: "signup" | "signin";
+}
+
+import { useState } from "react";
+import "../app/globals.css";
+import Link from "next/link";
+import { Button, buttonVariants } from "./ui/button";
+import { cn } from "../lib/utils";
+
+interface UserFormProps {
```

Comment:
why define the same props twice please remove this

Reply:
removed